src/hook_actions.ts

```diff
@@ -65,12 +65,22 @@ function conventionalHookRun(hook: Hook) {
 	}
 
 	vscode.window.showInformationMessage('Running ' + hook.label);
-
+	let command = 'sh';
+	var data = fs.readFileSync(hook.path, 'utf8');
+	const lines = data.split('\n');
+	for (let line of lines) {
+		if (line.startsWith('#!')) {
+			if (line.startsWith('#!/usr/bin/env')) {
```

Comment:
can make it `if (line.startsWith('#!') && line.startsWith('#!/usr/bin/env') )` to limit the if clauses

Reply:
When a line starts with `#!`, it is known as a shebang. When it further starts with `#!/usr/bin/env`, the content is parsed. Otherwise, it should exit the loop directly because the following content is invalid script statements.
`if (line.startsWith('#!') && line.startsWith('#!/usr/bin/env') )`  It will loop through all the statements